### DISCUSSION TITLE
Directory to save install.exe file

### DISCUSSION THREAD

**Discussion by dMaisnam** [JanDeDobbeleer/oh-my-posh#766] [Troubleshoot] [answered]
For oh-my-posh to work on Git bash:
- Should I save the `install.exe` file in `C:\Program Files\Git\bin` or `C:\Program Files\Git\usr\bin`? 
- Also, is the same directory where I added the executable the one I am supposed to add to PATH? 

Sorry if this sounds trivial. Thank you for your time.

**Comment by JanDeDobbeleer** [comment, marked as answer]
The installer contains the executable(s) and will add them to your PATH when you run it (double click). You can remove it once you ran it.

**Reply by dMaisnam** [reply]
@JanDeDobbeleer I was overcomplicating myself looking at an issue in this repo. It was very straightforward install. Thanks.
